src/routes/inngang/inngang-hovedkategorier.tsx

```diff
@@ -16,14 +16,17 @@ import {
 import Layout from '../layout';
 
 const InngangHovedkategorier = () => {
+    const title = 'Klage eller anke på vedtak';
     useLogPageView(PageIdentifier.INNGANG_HOVEDKATEGORIER);
+    document.title = title;
```

Comment:
I stedet for å skrive `document.title = <string>;` over alt, foreslår jeg å skille ut dette til `useDocumentTitle(title)`, som igjen bruker `useEffect()`.